identity-service/src/routes/healthCheck.js

```diff
@@ -10,35 +10,53 @@ const axios = require('axios')
 let notifDiscProv = config.get('notificationDiscoveryProvider')
 
 // Defaults used in relay health check endpoint
-const RELAY_HEALTH_ONE_HOUR_AGO_BLOCKS = 120 // 1 block/5sec = 120 blocks/hr
+const RELAY_HEALTH_TEN_MINS_AGO_BLOCKS = 120 // 1 block/5sec = 120 blocks/10 minutes
 const RELAY_HEALTH_MAX_TRANSACTIONS = 100 // max transactions to look into
 const RELAY_HEALTH_MAX_ERRORS = 5 // max acceptable errors for a 200 response
+const RELAY_HEALTH_MIN_NUM_USERS = 3 // min number of users affected to qualify for error
 const RELAY_HEALTH_MAX_BLOCK_RANGE = 200 // max block range allowed from query params
+// if (txs on blockchain / attempted) is less than this percent, health check will error
+// eg. 10 transactions on chain but 120 attempts should error
+const RELAY_HEALTH_SENT_VS_ATTEMPTED_THRESHOLD = 0.2
```

Comment:
@SidSethi we definitely could output this. the reason i picked this number as default (also can be overridden via env var) is because libs retries relays up to 5 times. basically the weight here. is determined by number of attempts taking into account retries and the number of successes. i set it lower rather than higher because in our cases the success txs count is 0, so this check will trigger. and i didn't want it to trigger on false positives if one person is retrying a failing tx over and over